crypto/srp/srp_vfy.c

```diff
@@ -574,21 +575,27 @@ SRP_user_pwd *SRP_VBASE_get1_by_user(SRP_VBASE *vb, char *username)
 
     if (RAND_priv_bytes(digv, SHA_DIGEST_LENGTH) <= 0)
```

Comment:
another case where we have multiple things with no libctx.. (and RAND_priv_bytes)

Reply:
`RAND_priv_bytes` doesn't need a libctx, the DRBGs are created once and then it is per thread.

Reply:
The point being that there are not too many instances of  RAND_priv_bytes() left..
i.e RAND_priv_bytes_ex() is used in most places.

Reply:
SRP interfaces are deprecated and we should not add _ex() calls to them.

Reply:
Hmmmm, no comment about the `EVP_MD_fetch(NULL, ..., NULL)` two lines down :laughing:  I can't see anywhere to extract a libctx from, so I'm with @t8m, live and let die.